code/src/components/Form.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+
+const languages = [' english', ' swedish', ' both'] // array for radio-buttons under language
+
+export const Form = (props) => {
+  const { genre, setGenre } = props
+  const { favorite, setFavorite } = props
+  const { language, setLanguage } = props
+  const { name, setName } = props
+  const { email, setEmail } = props
+
```

Comment:
It's not needed to have these divided. 
We can write: 
` const { genre, setGenre, `
`favorite, setFavorite,`
` language, setLanguage,  `
`name, setName, `
`email, setEmail } = props`